core/setup.py

```diff
@@ -64,7 +64,7 @@ def read(fname):
         'sqlparse>=0.2.3,<0.4',
         'networkx>=2.3,<3',
         'minimal-snowplow-tracker==0.0.2',
-        'colorama>=0.3.9,<0.4.4',
+        'colorama>=0.3.9,<0.4.5',
```

Comment:
We had to pin this before because the package didn't provide the source distribution on PyPI, but they have since added it! 👍 https://pypi.org/project/colorama/0.4.4/#files